### PR TITLE
[K4B-1773] Documentation for K4B partner association API (WIP)

### PR DESCRIPTION
**Pull request by kivra-ponern**

### CODE REVIEW

File: swagger.yml [hunk 1]
```diff
@@ -2887,7 +2887,101 @@ paths:
...
+          description: ''
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status: 
+                    description: Status of the signing process. One of `"pending"`, `"cancelled"` or `"complete"`. TODO
+                    type: string
+                    enum: ["pending", "cancelled", "complete"]
```
Comment (kivra-ponern) [inline swagger.yml:2967]: Not working.

File: swagger.yml [hunk 2]
```diff
@@ -2925,6 +3019,44 @@ paths:
...
+        - Enabling the interpretation service
+        - Adding the business mailbox to the partner's scope
+        - TODO (...)
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              required: [vat_number, bankid_order_key]
```
Comment (kivra-ponern) [inline swagger.yml:3044]: Not working.

File: swagger.yml [hunk 3]
```diff
@@ -2925,6 +3019,44 @@ paths:
...
+              properties:
+                vat_number:
+                  $ref: '#/components/schemas/VatNumber' 
+                bankid_order_key:
+                  type: string
+            example:
+              vat_number: SE556000475501
+              bankid_order_key: core_012345678901234567890123456789012345678901
```
Comment (kivra-ponern) [inline swagger.yml:3052]: Not sure if we wanna leak keys?

**Comment by kivra-fabber** [comment]
Closing this draft, needs more iteration first.

**Pull request closed without merging**